inventory/agents_test.go

```diff
@@ -15,6 +15,9 @@ import (
 	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
 )
 
+// AgentStatusUnknown means agent is not connected and we don't know anything about its status.
+var AgentStatusUnknown = "UNKNOWN"
```

Comment:
can we use `UNKNOWN` value from the API? since  your variable is not connected with API at the moment

Reply:
done